static/app/views/profiling/landing/slowestFunctionsTable.tsx

```diff
@@ -22,11 +26,31 @@ import useOrganization from 'sentry/utils/useOrganization';
 import useProjects from 'sentry/utils/useProjects';
 import {getPerformanceDuration} from 'sentry/views/performance/utils/getPerformanceDuration';
 
-import {ContentContainer, StatusContainer} from './styles';
+import {ContentContainer} from './styles';
 
 function sortFunctions(a: Profiling.FunctionMetric, b: Profiling.FunctionMetric) {
   return b.sum - a.sum;
 }
+function makeProfileLinkFromExample(
+  organization: Organization,
+  f: Profiling.FunctionMetric,
+  example: Profiling.FunctionMetric['examples'][0],
+  projectsLookupTable: Record<string, Project>
+) {
+  if (typeof example === 'string') {
+    return example;
```

Comment:
Is this right? Should if the `example` is a string, then it should be a profile id to link to right?

Reply:
I need to exclude this, it's referencing the type that mashes the old and the new references